src/csp.operations.js

```diff
@@ -754,7 +754,7 @@ function pipelineInternal(n, to, from, close, taskFn) {
   const jobs = chan(n);
   const results = chan(n);
 
-  times(n, () => {
+  range(n).forEach(() => {
```

Comment:
<img width="572" alt="screen shot 2017-02-08 at 9 01 06 pm" src="https://cloud.githubusercontent.com/assets/3093244/22740290/c0998120-ee41-11e6-82e7-2095acf6f4e2.png">

Can we do sth like this and get rid of times and range?